### PR TITLE
spellcheck

### PR DESCRIPTION
**Pull request by LPardue**

### CODE REVIEW

File: draft-ietf-quic-qlog-main-schema.md
Comment (lnicco) [inline draft-ietf-quic-qlog-main-schema.md:1146]: subtracting

File: draft-ietf-quic-qlog-main-schema.md
Comment (lnicco) [inline draft-ietf-quic-qlog-main-schema.md:108]: this kinda looks fine as it is? but I'll leave it to you, I don't feel strongly

File: draft-ietf-quic-qlog-quic-events.md
Comment (lnicco) [inline draft-ietf-quic-qlog-quic-events.md:1386]: great catch!

**Review by lnicco — APPROVED** [review]
found one thing to fix. approving to unblock you though. 
Feel free to merge after fixing the typo

File: draft-ietf-quic-qlog-quic-events.md
Comment (marten-seemann) [reply draft-ietf-quic-qlog-quic-events.md:1386]: oops

**Review by marten-seemann — APPROVED** [review]
lgtm, modulo @lnicco's comments

File: draft-ietf-quic-qlog-main-schema.md
Comment (LPardue) [reply draft-ietf-quic-qlog-main-schema.md:108]: This was my opinionated spellchecker, I'll revert.

**Pull request merged**
